packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+import {ICheckboxHandledProps, ICheckboxManagedClasses, ICheckboxUnhandledProps} from "./checkbox.props";
+import {ICheckboxClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+import {get} from "lodash-es";
+
+/* tslint:disable-next-line */
+class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasses, ICheckboxUnhandledProps, {}> {
+    protected handledProps: HandledProps<ICheckboxHandledProps & IManagedClasses<ICheckboxClassNameContract>> = {
+        managedClasses: void 0,
+        text: void 0,
+        checked: void 0,
+        indeterminate: void 0,
+        onChange: void 0
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLInputElement> {
+        return (
+            <label className={this.generateClassNames()}>
+                <input
+                    {...this.unhandledProps()}
+                    className={get(this.props, "managedClasses.checkbox_input")}
+                    type="checkbox"
+                    onChange={this.props.onChange}
+                    ref={this.setRef("items")}
+                />
+                <span className={get(this.props, "managedClasses.checkbox_label")}>
+                    {this.props.text}
+                </span>
+            </label>
+        );
+    }
+
+    /**
+     * Generates class names
+     */
+    protected generateClassNames(): string {
+        return super.generateClassNames(this.props.managedClasses.checkbox);
```

Comment:
you should also use `get` here to access the checkbox value of `managedClasses`

Reply:
@nicholasrice can you detail our purpose in using get from lodash here now? Looks like an updated principle for development.

Reply:
+1 I did some digging when I saw your usage in typography - also curious exactly why we are using lodash's get